docs/build/program/apis/data-client.md

```diff
@@ -29,12 +29,13 @@ Data client API methods are only available in the Python SDK.
 
 ## Establish a connection
 
-To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).
+To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate a [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).
```

Comment:
I think this sentence also needs to include the need for an API key and API key idea, since the next sentence tells us how to create one, but not why we need one. It becomes evident later why, but let's state it directly before we talk about getting it, maybe something like:

```suggestion
To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate a [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).

You will also need an API key and API key ID to authenticate your session.
To create a new API key, you have ... etc
```

Reply:
```suggestion
To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate a [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).

You will also need an API key and API key ID to authenticate your session.
To get an API key and ID, you have two options:
```